src/rollup/config.ts

```diff
@@ -365,7 +370,8 @@ export const plugins = [
             return { id: _resolved, external: false };
           }
         }
-        if (!resolved || resolved.external) {
+        // @ts-expect-error cast
+        if (!resolved || (resolved.external && !resolved.__nitro_external__)) {
```

Comment:
Is this `__nitro_external__` flag used other places?

Reply:
I forgot to update it, pushed